meeting-notes/notes-January-04-2021.md

```diff
@@ -0,0 +1,23 @@
+
+## Week of January 04, 2021
+
+### What can we talk about?
+
```

Comment:
Momentum: Thought exercise, not a criticism: How are you as a manager increasing tempo (think: how we work themes) while creating space for the people that are struggling cognitively?